pkgs/by-name/tl/tlock/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "tlock";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "eklairs";
+    repo = pname;
```

```suggestion
    repo = "tlock";
```
Altering `pname` should not change the final build result.